docs/work-with-us/releases/releases-and-versions.md

```diff
@@ -0,0 +1,46 @@
+# Leverage Product Releases
+Dear Leveragers,
+We're constantly kicking with a lot of improvements and some exciting new features
+
+## Reference Architecture
+
+!!! done ":calendar: RELEASES"
+    - [Releases | Reference Architecture for AWS](https://github.com/binbashar/le-tf-infra-aws/releases)
+    - [Releases | Reference Architecture for HCP Vault](https://github.com/binbashar/le-tf-vault/releases)
+
+## Leverage CLI
+
+!!! done ":calendar: RELEASES"
+    - [Releases | `leverage-cli`](https://github.com/binbashar/leverage)
+
+## Infrastructure as Code Library
+
+!!! done ":calendar: RELEASES"
+    ![leverage-tf](../../../assets/images/logos/terraform.png "Terraform"){: style="width:25px"} **Releases |Terraform Leverage™ Modules** :
+    
+    - [terraform-aws-waf-owasp](https://github.com/binbashar/terraform-aws-waf-owasp/releases)
+    - [terraform-aws-cost-billing-alarm](https://github.com/binbashar/terraform-aws-cost-billing-alarm/releases)
+    - [terraform-aws-vpc-flowlogs](https://github.com/binbashar/terraform-aws-vpc-flowlogs/releases)
+    - [terraform-aws-cost-budget](https://github.com/binbashar/terraform-aws-cost-budget/releases)
+    - [terraform-aws-tfstate-backend](https://github.com/binbashar/terraform-aws-tfstate-backend/releases)
+    - [terraform-aws-certbot-lambda](https://github.com/binbashar/terraform-aws-certbot-lambda/releases)
+    - [terraform-aws-ec2-basic-layout](https://github.com/binbashar/terraform-aws-ec2-basic-layout/releases)
+    - [terraform-aws-natgw-notifications](https://github.com/binbashar/terraform-aws-natgw-notifications/releases)
+    - [terraform-aws-guardduty-multiaccount](https://github.com/binbashar/terraform-aws-guardduty-multiaccount/releases)
+    - [terraform-aws-network-firewall](https://github.com/binbashar/terraform-aws-network-firewall/releases)
+    - [terraform-aws-backup-notifications](https://github.com/binbashar/terraform-aws-backup-notifications/releases)
+    - [terraform-aws-rds-export-to-s3](https://github.com/binbashar/terraform-aws-rds-export-to-s3/releases)
+    
+    ![leverage-tf](../../../assets/images/logos/terraform.png "Terraform"){: style="width:25px"} **Releases | Terraform Community Forks Modules**:
+    
+    - [terraform-aws-waf-owasp](https://github.com/binbashar/terraform-aws-waf-owasp/releases)
+    - ...
+    
+    ![leverage-helm](../../../assets/images/logos/helm.png "Terraform"){: style="width:25px"} **Releases | Helm Leverage™ Charts**:
+    
+    - [terraform-aws-sso](https://github.com/binbashar/terraform-aws-sso/tags)
```

Comment:
Should this module be listed under Helm Charts?

Reply:
@angelofenoglio no it won't, good catch :) I'll fix it right away.